.github/workflows/opensearch-sink-os-integration-tests.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         java: [11]
-        opensearch: [1.0.1, 1.1.0, 1.2.4, 1.3.0]
+        opensearch: [1.0.1, 1.1.0, 1.2.4, 1.3.1, 2.0.0]
```

Comment:
For what it's worth 1.3.2 is the most recent in the 1.3.x line - https://opensearch.org/lines/1x.html

Reply:
Thanks for noting that! I have pushed a new commit to use 1.3.2 instead.